Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM continuumio/miniconda3
+
+ARG QIIME2_RELEASE
+
+ENV LC_ALL C.UTF-8
+ENV LANG C.UTF-8
+ENV MPLBACKEND agg
+
+RUN conda install python=3.5
+RUN conda update -q -y conda
+RUN conda install --file https://data.qiime2.org/distro/core/qiime2-${QIIME2_RELEASE}-conda-linux-64.txt
+RUN qiime dev refresh-cache
+RUN echo "source tab-qiime" >> $HOME/.bashrc
```

Comment:
This does the same as before, only via the `.bashrc` route --- I think this is a lot more clear, and makes the `Dockerfile` pretty easy to understand.